core/src/test/java/mediumtest/hotspots/HotspotStatusChangeMediumTests.java

```diff
@@ -69,7 +69,6 @@ void tearDown() throws ExecutionException, InterruptedException {
     }
   }
 
-  @Test
```

Comment:
Any reason why this annotation has been removed? Also below in the file

Reply:
It's a mistake, because it has the same test name as the one on `IssueStatusChangeMediumTests`, and this one was failing, so I removed the wrong one (during our tests). Obviously, I'll revert it 😄